### PR TITLE
Lint all commits in a PR

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Had 10 minutes before lunch and @juliangieseke wanted to have each commit tested for a PR

```sh
[cos-ui-oss-pipeline_PR-2808-BOEJZQ7G7EXOOUCJZXKSP3XVN25NMZK4BHFHRIP5AADZEGLKJDYA] Running shell script

+ ./scripts/ci/lint-all-commits

fatal: ambiguous argument 'master..HEAD': unknown revision or path not in the working tree.

Use '--' to separate paths from revisions, like this:

'git <command> [<revision>...] -- [<file>...]'

script returned exit code 128
```

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: scripts/ci/lint-all-commits [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+GIT_BRANCH="${env.BRANCH_NAME}"
```
Comment (DanielMSchmidt) [inline scripts/ci/lint-all-commits]: @juliangieseke env.BRANCH_NAME is the jenkins naming, you would need to do `GIT_BRANCH=env.BRANCH_NAME ./scripts/ci/lint-all-commits` in the Jenkinsfile

**Comment by ahoskins** [comment]
I'm a bit confused why this is needed. Don't we lint commits when we commit them, and that's good enough?

**Comment by DanielMSchmidt** [comment]
@ahoskins I think @juliangieseke mentioned that only the last commit is looked at 🤔

**Comment by juliangieseke** [comment]
yup, wanted to check if we can lint not only the last commit in a PR. but all of them (to be able to generate our changelogs more reliable)

@DanielMSchmidt are you okay with closing this for now and picking this up in a later stage? Related ticket is already created: https://jira.mesosphere.com/browse/DCOS-21600

**Pull request closed without merging**